packages/terra-consumer-nav/src/components/nav-help/NavHelpContent.jsx

```diff
@@ -42,8 +42,8 @@ class NavHelpContent extends React.Component {
           <Arrange
             className={cx('help-item-text')}
             align="stretch"
-            fitStart={<div>{content.icon}</div>}
-            fill={<div className={cx('item-text-padding')}><SafeHtml text={content.text} /></div>}
+            fitStart={content.icon && <div className={cx('icon-text-padding')}>{content.icon}</div>}
```

Comment:
trick.. if there isn't an icon we can do this in css
```
.icon-text-padding:empty {
   display: none;
}
```
This lets us keep our JSX a little cleaner